packages/account/src/Sections/Security/TwoFactorAuthentication/__test__/digit-form.spec.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { WS } from '@deriv/shared';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import DigitForm from '../digit-form';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    WS: {
+        authorized: {
+            accountSecurity: jest.fn().mockResolvedValue({
+                error: { message: "That's not the right code. Please try again.", code: 'InvalidOTP' },
+            }),
+        },
+    },
+}));
+
+describe('<DigitForm />', () => {
+    const store = mockStore({
+        client: {
```

Comment:
we don’t need to specify all these fields. they exist by default.
check please mockStore implementation
as we use in this test case default values we can just do like this:
const store = mockStore({})

Reply:
okay